modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -133,6 +133,8 @@ Prior to Redpanda version 24.2, this meant that some cores on a broker could ina
 
 Starting in v24.2, topic-aware intra-broker partition balancing allows for dynamically reassigning partitions within a broker.  Redpanda prioritizes an even distribution of a topic's partition replicas across all cores in a broker. If a broker's core count changes, when the broker starts back up, Redpanda can check partition assignments across the broker's cores and reassign partitions, so that a balanced assignment is maintained across all cores. Redpanda can also check partition assignments when partitions are added to or removed from a broker, and rebalance the remaining partitions between cores.
 
+NOTE: Changing the number of CPU cores in a running cluster is supported from v24.2 only, and requires that you enable `node_local_core_assignment` as described in the previous note.
```

Comment:
Increasing has always been supported (but additional cores were initially idle).

Decreasing has not been supported until 24.2 (with the flag enabled).

Also, since 24.3 the flag will be enabled automatically.